eng/run-test.yml

```diff
@@ -0,0 +1,266 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  container: ''
+  testGroup: ''
+  readyToRun: false
+  helixQueues: ''
+  crossrootfsDir: ''
```

Comment:
unused as well :)

Reply:
Well, crossrootfsDir for sure. For helixQueues, that seems actively used in run-test. For readyToRun, that's likely unused. For testGroup, I was under the impression that is used as part of the test artifact name but I may be mistaken.

Reply:
Correct the diff is highlighting some variables I did not intend. I edited my comments to include the specific variables.

Reply:
Thanks Jarret for clarifying. I think I figured out some of the needed changes, could you please take another look when you have a chance to spot what else I need to take care of?

Reply:
In particular, please make sure I removed all the unneeded parameters, I was somewhat tricked by the incorrect highlighting into thinking all these parameters get somehow propagated automatically but, upon second thinking, such a prospect sounds quite ominous.